CONTRIBUTING.md

```diff
@@ -76,13 +91,16 @@ dotnet test OpenTelemetry.sln
 
 ### Code Coverage
 
-On Linux and macOS run `dotnet test` from a terminal window and you will see the following output:
+On Linux and macOS run `dotnet test` from a terminal window and you will see
+the following output:
 
 ![image](https://user-images.githubusercontent.com/20248180/59361025-1e1e7980-8d29-11e9-8449-548caf0d7823.png)
 
-Or, after running the tests, open the file `TestResults\Results\index.htm` in a browser.
+Or, after running the tests, open the file `TestResults\Results\index.htm` in a
```

Comment:
we don't export the html anymore. Maybe we should remove this section and add the command line to show the codecoverage